internal/compiler/builtins.slint

```diff
@@ -210,6 +210,7 @@ component ContextMenu inherits Empty {
     callback sub-menu(entry: MenuEntry) -> [MenuEntry];
     callback show(position: Point);
     function close() {}
+    function is-open() -> bool {}
```

Comment:
I think API wise I'd treat this as a property, not a function (like `Text.font-metrics`). But this is internal API :)